### PR TITLE
Adds import capability for clusters, database users, ip whitelist

### PR DESCRIPTION
**Pull request by Bowbaq**
@akshaykarle needed to import some stuff, figured I might as well contribute back.

### CODE REVIEW

File: mongodbatlas/database_user.go [hunk 1]
```diff
@@ -83,17 +88,17 @@ func resourceDatabaseUserCreate(d *schema.ResourceData, meta interface{}) error
...
-	d.Set("database", c.DatabaseName)
-	rolesMap := make([]map[string]interface{}, len(c.Roles))
-	for i, r := range c.Roles {
+	d.Set("username", u.Username)
+	d.Set("database", u.DatabaseName)
+	rolesMap := make([]map[string]interface{}, len(u.Roles))
+	for i, r := range u.Roles {
 		rolesMap[i] = map[string]interface{}{
-			"role":       r.RoleName,
+			"name":       r.RoleName,
```
Comment (Bowbaq) [inline mongodbatlas/database_user.go]: I think this was a bug, import wasn't working for roles and I couldn't figure out why until I checked the return value of `d.Set("roles", rolesMap)` which pointed out that `role` was not a valid key.

**Comment by akshaykarle** [comment]
Looks great! Thanks for the PR @Bowbaq :)

**Comment by Bowbaq** [comment]
@akshaykarle thanks for merging, would you mind cutting a release including these changes? thanks

**Comment by akshaykarle** [comment]
Sure, will do soon!

**Comment by akshaykarle** [comment]
Done, 0.6.1 is released!

**Pull request merged**